test/test_fcntl.rs

```diff
@@ -197,6 +199,72 @@ mod linux_android {
         let mut buf = [0u8; 200];
         assert_eq!(100, read(fd, &mut buf).unwrap());
     }
+
+    // This test is disabled for the target architectures below
+    // due to OFD locks not being available in the kernel/libc
+    // versions used in the CI environment.
```

Comment:
It's probably related to the fact that CI for these platforms runs under QEMU.

Reply:
I've amended the comment.